crates/core/src/commands/check.rs

```diff
@@ -71,18 +122,18 @@ impl CheckOptions {
 
                     let p = pb.progress_bytes(format!("checking {file_type:?} in cache..."));
                     // TODO: Make concurrency (20) customizable
-                    check_cache_files(20, cache, raw_be, file_type, &p)?;
+                    self.check_cache_files(20, cache, raw_be, file_type, &p)?;
```

Comment:
Not sure, if that could be just a field in `CheckOptions`? or maybe another struct `ParallelismOptions` composed into `CheckOptions` that we can use in rustic_core.